### PR TITLE
atd 1.2.0, atdgen 1.10.0

### PR DESCRIPTION
**Pull request by mjambon**
atdgen 1.10.0 depends on atd 1.2.0.

### CODE REVIEW

**Comment by avsm** [comment]
```
#=== ERROR while installing atd.1.2.0 =========================================#
# opam-version 1.2.2
# os           linux
# command      make
# path         /home/travis/.opam/4.03.0/build/atd.1.2.0
# compiler     4.03.0
# exit-code    2
# env-file     /home/travis/.opam/4.03.0/build/atd.1.2.0/atd-14020-736596.env
# stdout-file  /home/travis/.opam/4.03.0/build/atd.1.2.0/atd-14020-736596.out
# stderr-file  /home/travis/.opam/4.03.0/build/atd.1.2.0/atd-14020-736596.err
### stdout ###
# [...]
# ocamllex ag_doc_lexer.mll
# 46 states, 313 transitions, table size 1528 bytes
# make dep
# make[1]: Entering directory `/home/travis/.opam/4.03.0/build/atd.1.2.0'
# make[1]: `dep' is up to date.
# make[1]: Leaving directory `/home/travis/.opam/4.03.0/build/atd.1.2.0'
# make atdgen.cma atdgen.run
# make[1]: Entering directory `/home/travis/.opam/4.03.0/build/atd.1.2.0'
# ocamlfind ocamlc -dtypes -g -c -package "str atd biniou yojson" ag_version.ml
# make[1]: Leaving directory `/home/travis/.opam/4.03.0/build/atd.1.2.0'
### stderr ###
# ocamlfind: Package `atd' not found
# ocamlfind: Package `atd' not found
# make[1]: *** [ag_version.cmi] Error 2
# make: *** [all] Error 2
```

File: packages/atd/atd.1.2.0/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mjambon/atdgen/archive/v1.2.0.tar.gz"
```
Comment (dsheets) [inline packages/atd/atd.1.2.0/url]: I think this should be `https://github.com/mjambon/atd/archive/v1.2.0.tar.gz`.

Comment (avsm) [inline packages/atd/atd.1.2.0/url]: testing fix in #7423

Comment (mjambon) [inline packages/atd/atd.1.2.0/url]: Thanks!

**Pull request merged**